packages/alist/build.sh

```diff
@@ -2,13 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://alist.nn.ci
 TERMUX_PKG_DESCRIPTION="A file list program that supports multiple storage"
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION="3.37.1"
+TERMUX_PKG_VERSION="3.37.4"
+_ALIST_WEB_VERSION="3.37.1"
 TERMUX_PKG_SRCURL=(
 	https://github.com/alist-org/alist/archive/v${TERMUX_PKG_VERSION}.tar.gz
-	https://github.com/alist-org/alist-web/releases/download/${TERMUX_PKG_VERSION}/dist.tar.gz
+	https://github.com/alist-org/alist-web/releases/download/${_ALIST_WEB_VERSION}/dist.tar.gz
 )
 TERMUX_PKG_SHA256=(
-	528a967fc26d3d74c33c4efc2baa8cf5a64e82c167c582e749724fcefae10ce5
+	0f2c54aba7ddcfd7df3e959ce40709bbd88bf4eb9b0436079db879c8eb25b51b
 	0ca1fceb5291c59d5b5dcc0f3e871eac96485d39abce8495f1eb510cc12bccf6
 )
 TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
`alist` uses a stable release of alist web, but it doesn't provide web version with the source. If auto-update is enabled, the `ALIST_WEB_VERSION` should also be bumped with `TERMUX_PKG_REVISION`. `TERMUX_PKG_AUTO_UPDATE` should be set to false if `ALIST_WEB_VERSION` is introduced.